corehq/apps/users/migrations/0004_rm_role_id_from_admins.py

```diff
@@ -14,13 +14,15 @@ def fix_users(apps, schema_editor):
 
 
 def _get_admins_with_roles():
```

Comment:
You can just delete the whole `RunPython` operation for this migration - it's not relevant anymore

Reply:
Sure! This would be a no-op migration, then, yeah? I guess I will keep the code around just in case?

Reply:
To avoid confusing the migrations framework, the migration file still needs to be there, but you can just set `operations = []` like in this file:

https://github.com/dimagi/commcare-hq/blob/d9e67f9fde452ac5b3682af839f5b4583404436e/corehq/apps/users/migrations/0010_populate_usersinvitation.py